README.md

```diff
@@ -1,27 +1,63 @@
 # LogCheck API: Getting Started
 
-## General
+## Prerequisites
+- A **terminal program** (e.g. Terminal, iTerm) running a **sh-compatible shell** (e.g. Bash, ZSH)
+- A **scripting language**. Our examples use Python and Ruby, but you may use any scripting language that
+    supports making an HTTP GET requests with basic HTTP authentication.
+
+## Getting Started
+
+### 1. Generate credentials
 - Email support to request API access
```

Comment:
This shouldn't be so vague. @jpendry did we decide on how requests should be handled? Should they email help@logcheck.com or did you set up a special purpose group/alias?

Reply:
There is a [Dev Support](https://groups.google.com/a/logcheck.com/forum/#!forum/dev-support) list/group. The only thing I don’t like about that name is that some email responses from the group show up with a name of “Dev” instead of “Dev Support”, but I suspect there’s a setting to fix that somewhere.

Shouldn’t we be pointing to the same form that we point users to in the [Authentication](https://www.logcheckapp.com/developer#section/Authentication) section of our documentation? And if we’re not using that form anymore, we should make sure to update that section of the documentation.